runtime/gc_modron_standard/StandardAccessBarrier.hpp

```diff
@@ -92,6 +99,28 @@ class MM_StandardAccessBarrier : public MM_ObjectAccessBarrier
 	virtual bool preMonitorTableSlotRead(J9JavaVM *vm, j9object_t *srcAddress);	
 #endif	
 
+	/* New methods */
+	bool preObjectStoreInternal(J9VMThread *vmThread, J9Object *destObject, fj9object_t *destAddress, J9Object *value, bool isVolatile);
```

Comment:
align the naming... there is already postObjectStoreImpl, so this should be probably preObjectStoreImpl? although throughout GC code we then more often to use ...Internal than ...Impl. do a quick check what is more common before changing.

Reply:
renamed methods to preObjectStoreImpl.